src/set_theory/surreal.lean

```diff
@@ -66,6 +66,123 @@ end
 
 instance : has_mul pgame := ⟨mul⟩
 
+/-- An explicit description of the moves for Left in `x * y`. -/
+def left_moves_mul {x y : pgame} : (x * y).left_moves
+  ≃ x.left_moves × y.left_moves ⊕ x.right_moves × y.right_moves :=
+by { cases x, cases y, refl, }
+
+/-- An explicit description of the moves for Right in `x * y`. -/
+def right_moves_mul {x y : pgame} : (x * y).right_moves
+  ≃ x.left_moves × y.right_moves ⊕ x.right_moves × y.left_moves :=
+by { cases x, cases y, refl, }
+
+@[simp] lemma mk_mul_move_left_inl {xl xr yl yr} {xL xR yL yR} {i j} :
+  (mk xl xr xL xR * mk yl yr yL yR).move_left (sum.inl (i,j))
+  = xL i * (mk yl yr yL yR) + (mk xl xr xL xR) * yL j - xL i * yL j
+:= rfl
```

Comment:
A few style points:
```suggestion
  (mk xl xr xL xR * mk yl yr yL yR).move_left (sum.inl (i, j))
    = xL i * (mk yl yr yL yR) + (mk xl xr xL xR) * yL j - xL i * yL j :=
rfl
```
`:=` always goes at the end of a line, and `,` I think is usually followed by a comma. Same below.

Reply:
Ah, sorry about that. 

Fixed it now.